EOLib/PacketHandlers/Avatar/AvatarReplyHandler.cs

```diff
@@ -0,0 +1,80 @@
+﻿using AutomaticTypeMapper;
+using EOLib.Domain.Character;
+using EOLib.Domain.Extensions;
+using EOLib.Domain.Map;
+using EOLib.Domain.Notifiers;
+using EOLib.Net;
+using EOLib.Net.Handlers;
+using EOLib.Domain.Login;
+using System.Collections.Generic;
+using System;
+
+namespace EOLib.PacketHandlers.Avatar
+{
+    [AutoMappedType]
+    public class AvatarReplyHandler : InGameOnlyPacketHandler
+    {
+        private readonly ICurrentMapStateRepository _currentMapStateRepository;
+        private readonly IEnumerable<IOtherCharacterAnimationNotifier> _otherCharacterAnimationNotifiers;
+        private readonly IEnumerable<IOtherCharacterEventNotifier> _otherCharacterEventNotifiers;
+        private readonly IEnumerable<IMainCharacterEventNotifier> _mainCharacterEventNotifiers;
+        private readonly ICharacterRepository _characterRepository;
+
+        public AvatarReplyHandler(
+            IPlayerInfoProvider playerInfoProvider,
+            ICurrentMapStateRepository currentMapStateRepository,
+            ICharacterRepository characterRepository,
+            IEnumerable<IOtherCharacterAnimationNotifier> otherCharacterAnimationNotifiers,
+            IEnumerable<IMainCharacterEventNotifier> mainCharacterEventNotifiers,
+            IEnumerable<IOtherCharacterEventNotifier> otherCharacterEventNotifiers)
+            : base(playerInfoProvider)
+        {
+            _currentMapStateRepository = currentMapStateRepository;
+            _characterRepository = characterRepository;
+            _otherCharacterAnimationNotifiers = otherCharacterAnimationNotifiers;
+            _mainCharacterEventNotifiers = mainCharacterEventNotifiers;
+            _otherCharacterEventNotifiers = otherCharacterEventNotifiers;
+        }
+
+        public override PacketFamily Family => PacketFamily.Avatar;
```

Comment:
Nit - put the Family/Action after readonly fields and before constructor.

Additional nit - make sure there's a line of whitespace between publicly visible members.